cmd/kubeadm/app/constants/constants_test.go

```diff
@@ -110,3 +113,43 @@ func TestAddSelfHostedPrefix(t *testing.T) {
 		}
 	}
 }
+
+func TestEtcdSupportedVersion(t *testing.T) {
+	var tests = []struct {
+		kubernetesVersion string
+		expectedVersion   *version.Version
+		expectedError     error
+	}{
+		{
+			kubernetesVersion: "1.8.0",
```

Comment:
nit: add test cases for `"v1.9.0"`,  `"v1.8.6"` and `"v1.10.0"` as well?

Reply:
done